docs/data/data-grid/layout/layout.md

```diff
@@ -3,9 +3,8 @@
 <p class="description">The Data Grid offers multiple layout modes.</p>
 
 :::error
-By default, the Data Grid has **no intrinsic dimensions**.
-Instead, it takes up the space given by its parent.
-The Data Grid will raise an error in the console if its container has no intrinsic dimensions.
```

Comment:
Not true [due to this](https://mui-org.slack.com/archives/C05SBSU8NPR/p1756413803981159). If we want the check, we need to rework it and it would be a different rendering sequence in dev than in prod. I don't have a strong opinion, I'd prefer to have feedback from the @mui/xgrid team for how to proceed.